tests/python/unittest/test_higher_order_grad.py

```diff
@@ -210,6 +217,168 @@ def check_second_order_unary(x, op, grad_grad_op, rtol=None, atol=None):
                         x.grad.asnumpy(), rtol=rtol, atol=atol)
 
 
+def arange_shape_like(y):
+    shape = y.shape
+    nelems = reduce(mul, shape)
+    x = nd.arange(nelems).reshape(shape)
+    return x
+
+
+class NDArrayGenerator(object):
```

Comment:
Can we use existing test util instead of creating new ones?

Reply:
I don't see the issue.